src/app/plugins/addclasschans.plugin.ts

```diff
@@ -23,7 +23,7 @@ export default class AddClassChannelsPlugin extends Plugin {
   private _CATEGORIES: string[] = ['cs', 'it', 'ee', 'csgrad', 'eegrad', 'gened'];
   private _NEW_CHAN_MESSAGE =
     'Welcome to the class!\n\n' +
-    '**If it has not been done so already, please post the #class_invite ' +
+    '**If it has not been done so already, please post the [placeholder] ' +
```

Comment:
Put this in a method so that you can use a format string instead of relying on substitution

Reply:
Would you like `_NEW_CHAN_MESSAGE` to be declared in a method? Since we are fetching the channel ID, it would require an async method which seems messy.

Reply:
```
private _getNewChanMessage(classInviteChanId) {
  return `.... ${classInviteChanID} ...`
}
```